react/week3/src/components/Todo.js

```diff
@@ -0,0 +1,63 @@
+import React , { useState } from 'react'
+
+function Todo({ todo, index, completeTodo, removeTodo, updateTodo }) {
+ 
+  const [editTask, seteditTask] = useState(false);

```

Comment:
This needs to be like this instead
```suggestion
  const [editTask, setEditTask] = useState(false);
```

Reply:
Also, this should be named as a flag. `isEditMode`, with the current naming we don't know if it's a boolean or not.
```suggestion
  const [isEditMode, setIsEditMode] = useState(false);

```